data/json/itemgroups/forage.json

```diff
@@ -4,64 +4,83 @@
+      { "item": "egg_reptile", "prob": 15, "count-min": 1, "count-max": 3 },
+      { "item": "wild_herbs", "prob": 60 },
+      { "item": "blackberries", "prob": 60, "count-min": 2, "count-max": 5 },
+      { "item": "buckwheat", "prob": 10, "count-min": 1, "count-max": 4 },
+      { "item": "thyme", "prob": 10, "count-min": 1, "count-max": 3 },
+      { "item": "garlic", "prob": 2 },
+      { "item": "seed_garlic", "prob": 8, "count-min": 1, "count-max": 6 },
+      { "item": "seed_canola", "prob": 20, "count-min": 1, "count-max": 3 },
+      { "item": "dogbane", "prob": 20, "count-min": 1, "count-max": 3 },
+      { "item": "bee_balm", "prob": 20, "count-min": 1, "count-max": 3 },
+      { "item": "mugwort", "prob": 20, "count-min": 1, "count-max": 3 },
+      { "item": "chive", "prob": 10, "count-min": 1, "count-max": 2 },
+      { "item": "oregano", "prob": 10, "count-min": 1, "count-max": 2 },
+      { "item": "cilantro", "prob": 10, "count-min": 1, "count-max": 2 },
+      { "item": "dill", "prob": 10, "count-min": 1, "count-max": 2 }
     ]
   },
   {
     "id": "forage_autumn",
     "type": "item_group",
     "subtype": "distribution",
     "entries": [
-      { "group": "forage_mushroom", "prob": 50 },
-      { "item": "wild_herbs", "prob": 10 },
-      { "item": "acorns", "prob": 100, "count-min": 2, "count-max": 3 },
+      { "group": "forage_mushroom", "prob": 150 },
+      { "item": "wild_herbs", "prob": 30 },
+      { "item": "acorns", "prob": 300, "count-min": 2, "count-max": 3 },
       { "item": "buckwheat", "prob": 15, "count-min": 2, "count-max": 4 },
       { "item": "thyme", "prob": 5, "count-min": 1, "count-max": 3 },
       { "item": "garlic", "prob": 1 },
       { "item": "seed_garlic", "prob": 4, "count-min": 1, "count-max": 6 },
       { "item": "seed_canola", "prob": 5, "count-min": 1, "count-max": 3 },
       { "item": "dogbane", "prob": 5, "count-min": 1, "count-max": 3 },
       { "item": "bee_balm", "prob": 5, "count-min": 1, "count-max": 3 },
-      { "item": "mugwort", "prob": 5, "count-min": 1, "count-max": 3 }
+      { "item": "mugwort", "prob": 5, "count-min": 1, "count-max": 3 },
+      { "item": "chive", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "oregano", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "cilantro", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "dill", "prob": 5, "count-min": 1, "count-max": 2 }
     ]
   },
   {
     "id": "forage_winter",
     "type": "item_group",
     "subtype": "distribution",
-    "entries": [  ]
+    "entries": [
+      { "item": "wild_herbs", "prob": 30 },
+      { "item": "acorns", "prob": 30, "count-min": 2, "count-max": 3 },
+      { "item": "garlic", "prob": 1 },
+      { "item": "seed_garlic", "prob": 4, "count-min": 1, "count-max": 6 },
+      { "item": "hickory_nut", "prob": 6, "count-min": 1, "count-max": 3 },
+      { "item": "cranberries", "prob": 8, "count-min": 2, "count-max": 6 }
```

Comment:
Actually cranberries I think should be removed here entirely, since shinino has made cranberries have their own bush which has its own harvest.  Foraging item groups are for finding things in the underbrush.

Reply:
So, a few things here.  You're right that cranberry (vines) are harvested in fall.  However, that's when man harvests them.  It doesn't mean that they suddenly vanish after that.  I went by the following link, which lists them as being able to be found via foraging in New England during winter -- granted, this would probably be early winter versus late winter, but we don't have a way to do monthly foraging tables.

https://joshfecteau.com/resources/wild-edibles-monthly-guide/#toc-8

I'll admit, a lot of the confusion here may be on my side.  For example, I was under the impression that things in furniture.json were survivor-grown things, while terrain.json was naturally-occurring.   This is why I put the cranberry bush in terrain.json, because there wasn't a way to naturally find the bushes.  If it should only be in terrain -or- foraging, but not both, that's fine, and I can live with that.

Reply:
If you added a cranberry bush, which is fully justified in my opinion, I'd follow @cosmiccoincidence advice and drop listing them in the foraging table altogether. Personaly I'd also keep the consistency and make them like other berry bushes.

Reply:
I'll update this later today.  I'll chalk it up to me misunderstanding things. :)